devolving_music/lib/song_scores.py

```diff
@@ -31,10 +32,12 @@ def get_scores(self) -> "dict[int, ScoreSuite]":
 
     def get_compare_submission_random(self, submission_id) -> SongSubmission:
         key_list = list(self.song_score_dict.keys())
-        key_list.remove(submission_id)
+        if(submission_id > 0):
```

Comment:
What's the intent of this condition?

Reply:
if you want to not remove the id ie you wanted any random song you would just pass in -1 to submission id